libfuse/fs.go

```diff
@@ -199,18 +202,22 @@ func (f *FS) Root() (fs.Node, error) {
 
 // Statfs implements the fs.FSStatfser interface for FS.
 func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
-	// TODO: Fill in real values for these.
-	var bsize uint32 = 32 * 1024
 	*resp = fuse.StatfsResponse{
-		Blocks:  ^uint64(0) / uint64(bsize),
-		Bfree:   ^uint64(0) / uint64(bsize),
-		Bavail:  ^uint64(0) / uint64(bsize),
-		Files:   0,
-		Ffree:   0,
-		Bsize:   bsize,
+		Bsize:   fuseBlockSize,
 		Namelen: ^uint32(0),
-		Frsize:  0,
+		Frsize:  fuseBlockSize,
+	}
+	usageBytes, limitBytes, err := f.quotaUsage.Get(ctx, QuotaUsageStaleTolerance)
+	if err != nil {
+		return fuse.EIO
```

Comment:
> Would you prefer to call reportErr here or is it overkill?

Nah, we removed statfs logging for the happy path since it caused the dsl fuse tests to log too much.